packages/backend/server/src/plugins/copilot/context/job.ts

```diff
@@ -100,11 +114,8 @@ export class CopilotContextDocJob {
     });
   }
 
-  // @OnEvent('doc.indexer.deleted')
-  async deleteDocEmbeddingQueueFromEvent(
-    // TODO(@darkskygit): replace this with real event type
-    doc: { workspaceId: string; docId: string } //Events['doc.indexer.deleted'],
-  ) {
+  @OnEvent('doc.indexer.deleted')
+  async deleteDocEmbeddingQueueFromEvent(doc: Events['doc.indexer.deleted']) {
     await this.models.copilotContext.deleteWorkspaceEmbedding(
```

Comment:
Here, it is also necessary to determine `supportEmbedding`

Reply:
done